src/Redmine/Client.php

```diff
@@ -105,6 +105,16 @@ class Client
      */
     private $responseCode = null;
 
+    /**
+     * @var string Redmine response content type
+     */
+    private $responseContentType = '';
```

Comment:
```suggestion
    private string $responseContentType = '';
```
how about trying to type every (new) property?

Reply:
This would be require PHP 7.4, but I tried to match PHP 7.3 as well.